### PR TITLE
feat: add Avalanche Fuji testnet support [APE-705]

### PR DESCRIPTION
**Pull request by PoCk3T-SPAI**
### What I did

As per https://docs.snowtrace.io/v/fuji-snowtrace, adding the Etherscan/Snowtrace API for Avalanche Fuji testnet in client.py

### Checklist

- [X] Passes all linting checks (pre-commit and CI jobs)
- [X] New test cases have been added and are passing
- [X] Documentation has been updated
- [X] PR title follows [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) standard (will be automatically included in the changelog)

### CODE REVIEW

**Comment by PoCk3T-SPAI** [comment]
@fubuloubu humble contribution here
I've identified 4 files where additions needed to be made to support the Fuji testnet of Avalanche
Hope I didn't miss to update any other file
Let me know your thoughts

**Review by fubuloubu — DISMISSED** [review]

**Comment by PoCk3T-SPAI** [comment]
@fubuloubu thanks for the prompt action; 
2 CI/CD tests seem to have failed:
- Lint -> not sure what's the hold up, I followed the exact same syntax as the other ecosystems/chains listed and the lint test logs are not very descriptive
- Functional test -> it seems to be a test involving connecting to the ethereum mainnet, so out of scope for this PR
What do you think? Any way you could force-merge this PR? Also happy to try and correct anything if needed?

**Comment by fubuloubu** [comment]
> @fubuloubu thanks for the prompt action; 
> 2 CI/CD tests seem to have failed:
> - Lint -> not sure what's the hold up, I followed the exact same syntax as the other ecosystems/chains listed and the lint test logs are not very descriptive

To fix the lint issue, just run `black` or install `pre-commit`

**Comment by fubuloubu** [comment]
For the tests, you are right, they won't execute for you specifically because you don't have a Vess to our secrets. If you can run the new test locally (just grab an etherscan key) we can merge it with the lint fix and be on our way

**Comment by PoCk3T-SPAI** [comment]
Thanks @fubuloubu , done, hope it works!

File: ape_etherscan/client.py [hunk 1]
```diff
@@ -92,7 +92,11 @@ def get_etherscan_api_uri(ecosystem_name: str, network_name: str):
             else "https://api-testnet.polygonscan.com/api"
```
Comment (unparalleled-js) [inline ape_etherscan/client.py:92]: In line 50 of this file, we need to add this one `https://testnet.snowtrace.io/`, right?

Comment (unparalleled-js) [reply ape_etherscan/client.py:92]: You added the API one, but we also need the txn URL one

Comment (PoCk3T-SPAI) [reply ape_etherscan/client.py:92]: I think the transaction URL is generated automatically: https://github.com/ApeWorX/ape-etherscan/blob/main/ape_etherscan/explorer.py#L25

But I agree, https://github.com/ApeWorX/ape-etherscan/blob/main/ape_etherscan/client.py#L50 needs extra logic to refer to https://testnet.snowtrace.io/ for Fuji testnet

Comment (PoCk3T-SPAI) [reply ape_etherscan/client.py:92]: See https://github.com/ApeWorX/ape-etherscan/pull/75

Comment (unparalleled-js) [reply ape_etherscan/client.py:92]: thanks, this is exactly what I meant!

**Pull request merged**